R/util_scale_computation.R

```diff
@@ -18,7 +18,7 @@ sc.append_scales <- function(
     # filter any non-scale variables from scale_variables_table
     scale_variables_table <- filter(
         scale_variables_table,
-        !is.na(scale)
+        !util.is_blank(scale)
```

Comment:
Just to double check: are there never cases where a "blank" value, which would be newly filtered by `util.is_blank()`, would count as information you _don't_ want to filter? Like an empty string?

Reply:
I can't imagine there being empty string values, spaces, etc. that we'd want to retain in this context.